src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -231,21 +235,28 @@ public Conference(Videobridge videobridge,
         this.eventAdmin = enableLogging ? videobridge.getEventAdmin() : null;
         this.includeInStatistics = enableLogging;
         this.name = name;
-        this.logPrefix = "[id=" + id + " gid=" + gid + " name=" + name + "] ";
-
-        if (!enableLogging)
+//        this.logPrefix = "[id=" + id + " gid=" + gid + " name=" + name + "] ";
+        Map<String, String> context = new HashMap<>();
+        context.put("confId", id);
+        context.put("gId", gid);
+        if (name != null)
         {
-            logger.setLevel(Level.WARNING);
+            context.put("name", name.toString());
         }
 
+        Level minLevel = enableLogging ? Level.ALL : Level.OFF;
```

Comment:
I think this should be WARN or ERROR for health check conferences

Reply:
yup you're right, will fix.